common/pm/pm.cpp

```diff
@@ -0,0 +1,51 @@
+#include "pm.hpp"
+
+namespace {
+
+constexpr u64 QLAUNCH_TITLE_ID{0x0100000000001000ULL};
+u64 CURRENT_TITLE_ID{};
+
+} // namespace
+
+namespace pm {
+
+auto Initialize() -> Result {
+    auto rc = pmdmntInitialize();
```

Comment:
Should use pgl on 10.0.0+ imo

Reply:
I didn't know that was a thing. I'll check it out

Reply:
is there any reason to favour pgl on 10+? wouldn't having a single impl that works on all fw easier to maintain?

Reply:
huh I thought pgl had more session than pm:dmnt. 
Nevermind then